src/components/login-card/style.less

```diff
@@ -2,32 +2,30 @@
 @import '../../styles/fonts.css';
 @import '../../styles/mixins.less';
 
-#card{
-  width: 35vw;
-  height: 60vh;
+.card-login{
+  width: 80vw;
```

Comment:
Can you add the card widths/height into the ./styles vars file and import it from there so we don't have magic numbers